src/controller/artifact/processor/default_test.go

```diff
@@ -116,6 +116,31 @@ var (
         }
     ]
 }`
+	v2ManifestWithUnknownConfig = `{
+    "schemaVersion": 2,
+    "mediaType": "application/vnd.oci.image.manifest.v1+json",
+    "config": {
+        "mediaType": "application/vnd.nhl.peanut.butter.bagel",
+        "digest": "sha256:ee29d2e91da0e5dbf6536f5b369148a83ef59b0ce96e49da65dd6c25eb1fa44f",
+        "size": 33,
+        "newUnspecifiedField": null
+    },
+    "layers": [
+        {
+            "mediaType": "application/vnd.oci.empty.v1+json",
+            "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+            "size": 2,
+            "newUnspecifiedField": "null"
+        }
+    ],
+    "subject": {
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "digest": "sha256:5a01bbc4ce6f52541cbc7e6af4b22bb107991a4bdd433103ff65aeb00756e906",
+        "size": 714,
+        "newUnspecifiedField": null
+    }
+ }`
+	unknownConfig = `{NHL Peanut Butter on my NHL bagel}`
```

Comment:
variable name change to `unknownConfigData`? 
`{NHL Peanut Butter on my NHL bagel}` => `NHL Peanut Butter on my NHL bagel` ?

Reply:
Followed the name convention of this UT. Such as `ormbConfig`